data/seeds/04_photo.js

```diff
@@ -1,156 +0,0 @@
-exports.seed = function (knex) {
```

Comment:
Do you think we could include these seeds for the existing code, just for the time being? We can remove them if they wind up causing conflicts as our code evolves.